requirements.txt

```diff
@@ -1,7 +1,8 @@
 pip~=22.0
 ninja
 cmake
-cuquantum>=22.3.0
+cuquantum-cu11
```

Comment:
```suggestion
custatevec-cu11
```

Reply:
I put cuquantum-cu11 here is to aligned with the context of readme file, which mentions nvidia cuquantum sdk. let's update it with custatevec-cu11 here.